src/pages/workspace/taxes/WorkspaceCreateTaxPage.tsx

```diff
@@ -84,6 +86,7 @@ function WorkspaceCreateTaxPage({
                             submitButtonText={translate('common.save')}
                             enabledWhenOffline
                             shouldValidateOnBlur={false}
+                            disablePressOnEnter={!!modal?.isVisible}
```

Comment:
`modal?.isVisible` is also true if navigation stack modal is visible, which caused this issue:

https://github.com/Expensify/App/issues/66723